### PR TITLE
Reverts #8900 and #8897

### PR DESCRIPTION
**Pull request by axietheaxolotl**
## About The Pull Request

cab

See title. Salt PR time.

Reverts #8900 and #8897.

## How This Contributes To The Skyrat Roleplay Experience

Oversized is not a good quirk, I'm gonna say that right here and now. I dislike this quirk immensely, even if only for it's premise, however, the PR made is not good enough in it's current quality, and makes me dislike it even more.

Oversized's only reason for existence is for fursona bullshit, and I can understand that we're a furry server. But this is just not a good addition to compliment that.

One, oversized sprites look awful, ignoring sprite scaling and how that can fuck things up, just the fact of there are sprites that are 200% size just doesn't look good. It takes away from immersion, looks bad, and just ruins the environment.

Two, in its most basic sense, this is a powergaming perk. Security, as Goofball mentioned in the original PR, can now just, not worry about grabs, because I have le funny big size. The downsides given to mitigate this are so inconsequential they aren't even worth mentioning. The only good one is the slowdown, which even then, is not that much of a slowdown, considering people do that voluntarily already by giving duffle bags, which provide the same amount of debuff speed. It was suggested to give them chunky fingers, but this was denied by the person who made the request because they quote: Want to look threatening. Which is another part of the power gaming dynamic, where people think big size = intimidating. No, this is not true, but, it's worth mentioning that a reason the person wanted this was so they could have a direct advantage over others.

tl;dr, the benefits that oversized give **vastly** overweigh its downsides, so it needs to be much more nerfed.

Third, this is hypercode tier stuff, to be honest. We, for a specific reason, didn't allow people to go over 150% sprite scaling. Because it's not good, it just looks bad. I said this stuff before, but I'll elaborate further. This is a problem I believe Skyrat in general suffers from, a noticeable example is le furry rainbow vomit dogs that populate the server. They're not good, they take out of the immersion, and ruin the environment when there's some bright blue dog running around. Oversized has this same issue and exaggerates it more by making it mechanical. If this was entirely cosmetic, I wouldn't care that much, but we've decided to make this mechanical for some god-forsake reason.

Also, this was merged ridiculous quickly, to where I thought it was an FM. It took five hours to go from PR, to merged. This is not an adequate amount of time to provide discussion and people's thoughts.

## Changelog

:cl:
del: Reverted and removed the oversized quirk
/:cl:

### CODE REVIEW

**Comment by Iamgoofball** [comment]
They merged it due to some issues with merge conflicts, we were trying to figure out why goofsec was conflicting when we tried to put it back on the server and it came down to a stealthy .dme conflict between Oversized and GoofSec.

Sorry!

**Comment by axietheaxolotl** [comment]
> They merged it due to some issues with merge conflicts, we were trying to figure out why goofsec was conflicting when we tried to put it back on the server and it came down to a stealthy .dme conflict between Oversized and GoofSec.

guh

**Comment by axietheaxolotl** [comment]
I hate CI

**Comment by softcerv** [comment]
Personally I think that it's good for the server to have a perk like this because it makes playing a big character more of a unique experience, but I can agree with you that this should have undergone more work before being rolled out to the server. There is also some improvements that I could see being made to the balancing around it, but I'd much rather get into that on the original PR or on discord instead of here.

**Comment by SpaceVampire** [comment]
> They merged it due to some issues with merge conflicts, we were trying to figure out why goofsec was conflicting when we tried to put it back on the server and it came down to a stealthy .dme conflict between Oversized and GoofSec.

I'm a bit confused about why it would have been full merged right away, instead of just not test-merged until the issue was figured out. Given that it should have still be under discussion. But this might be outside the scope of this PR.

Oh on another note #8900 and #8897, should probably be mentioned in the PR body for easy cross-linking.

**Comment by axietheaxolotl** [comment]
> Oh on another note #8900 and #8897, should probably be mentioned in the PR body for easy cross-linking.

donezo

File: code/modules/mob/living/carbon/carbon_defense.dm [hunk 1]
```diff
@@ -498,25 +498,20 @@
...
-
-			//SKYRAT EDIT ADDITION BEGIN - EMOTES
-			if(HAS_TRAIT(src, TRAIT_EXCITABLE))
-				if(!src.dna.species.is_wagging_tail(src))
-					src.emote("wag")
-			//SKYRAT EDIT ADDITION END
-
-			if(HAS_TRAIT(src, TRAIT_BADTOUCH))
-				to_chat(M, span_warning("[src] looks visibly upset as you pat [p_them()] on the head."))
+	SEND_SIGNAL(src, COMSIG_CARBON_HEADPAT, M)
```
Comment (Seris02) [inline code/modules/mob/living/carbon/carbon_defense.dm]: you forgot an indent

**Comment by Krysonism** [comment]
Having any kind of sprite scaling is a bruh moment when it comes to sprite quality.

**Comment by Arctaisia** [comment]
The reason we didn't go over 150% sprite scaling was due to balance reasons, it's been like that since oldbase. We would only have added in 200% sprite size if it had mechanical debuffs and buffs that it would give, which is why this one was made. We didn't not add it because sprite scaling wasn't working with it. Personally, I don't see much issue with it. It's more customisation, and yes, people are tall, but it isn't 300% like I've seen on other servers. There are alien species around and people 2x the size of normal won't be too uncommon. 
I do believe it should have been TM'd first, but personally, I don't feel that calling it 'hyperstation code' is really justified, seeing as every other furry server, from VOREcode to other CITcode servers, and even a downstream of Skyrats all have 200% without issue. It's just normal for furry servers to have customisation, and we should allow that power to be in the people's hands, but ofcourse, not too overboard, which is where these nerfs come in.
Weh.
Maybe rather than delete it, just add more nerfs, or remove the buffs, or remove all changes to the character stats (besides size) overall? You did state that in your PR, afterall. That way, there won't be any benefits outweighing negatives, and people can enjoy their 200% sprite characters. Afterall, 200% sprite size with no benefits or downsides just means a bigger hitbox, and easier to hit during combat.

**Comment by GoldenAlpharex** [comment]
> Hmmm, today I will open a salt PR and proceed to shit-talk the entire community in the PR body, surely this can only go well.

Furthermore, some more tweaks and adjustments for this quirk will be made, as it was merged with the intent to allow us to test it without having conflicts with another test-merge.

Lastly, this kind of attitude is not one we tolerate. GitHub is not the place to go and write a rant on why you think "rainbow vomit dogs" are ruining Skyrat. I can't believe I actually have to say this, but if you want to contribute to the repository, don't be toxic about it.

**Comment by axietheaxolotl** [comment]
> Furthermore, some more tweaks and adjustments for this quirk will be made, as it was merged with the intent to allow us to test it without having conflicts with another test-merge.

It should've waited until one of them was full merged, and gone the full process then this force merge.

> Lastly, this kind of attitude is not one we tolerate. GitHub is not the place to go and write a rant on why you think "rainbow vomit dogs" are ruining Skyrat. I can't believe I actually have to say this, but if you want to contribute to the repository, don't be toxic about it.

At no point did I ever personally insult anyone. Or attack the whole community. Nor did I say any one thing was "ruining Skyrat." I complained about immersion breaking, and the environment, which are entirely valid, personal points. You've strung this from thin air. I used the rainbow vomit dogs as an example and compared them to the big sprites. At no point did I say "People who play rainbow vomit dogs are ruining the game"

As well, you're attacking my character and language instead of actually focusing on the points I've made lol.

**Comment by gonenoculer5** [comment]
👍

**Comment by gonenoculer5** [comment]
arguably simpler solution is just to disable both, rework them, then re-enable them. Job done.

**Comment by hugmeorelse** [comment]
I did not approve the quirk, and I am looking to revert this. However, this behavior is unacceptable, @axietheaxolotl. We had this talk before, and I am starting to question whether or not if you are a good fit for this community. The Github is explicitly not for preaching. I have made this clear on numerous occasions.

**Comment by Trubbl3** [comment]
instead of forcing mechanical stuff i rather keep it all RP whise
